App.js

```diff
@@ -1,26 +1,109 @@
-import React from "react"
-import styled from "styled-components/native"
+import React, {useState, useEffect} from "react";
+import styled from "styled-components";
+import { Image, View, Text, TouchableOpacity, Share, Button, ScrollView} from 'react-native'
+import { startAsync } from "expo/build/AR";
+import { isRequired } from "react-native/Libraries/DeprecatedPropTypes/DeprecatedColorPropType";
+
 
-const App = () => {
-  return (
-    <Container>
-      <Title>This is your cool app!</Title>
-      <Title>Go to App.js and start coding</Title>
-      <Title>💅💅💅</Title>
-    </Container>
-  )
-}
+
+const apiKey = `fc072b13-28ea-4264-a23b-cd3fd71d22bd`
+const url = "https://api.thecatapi.com/v1/images/search?limit=10"
 
 const Container = styled.View`
-  flex: 1;
-  background-color: papayawhip;
+  flex:1;
+  background-color: black;
   justify-content: center;
-  align-items: center;
 `
 
 const Title = styled.Text`
-  font-size: 24px;
-  color: palevioletred;
+  background-color: black;
+  color: white;  
+  margin-top: 60;
+  margin-bottom: 10; 
+  margin-left: 0;
+  margin-right: 0;
+  padding: 50px;
+  justify-content: center;
+  align-items: center;
+  font-size: 25px;
+  text-align: left;
+  width: 300px;
+`
+
+const ImageCat = styled.Image`
+ flex: 1;
+ width: 100;
+ height: 100;
+ position: absolute;
+ left: 270;
+ top: 110;
 `
 
+const ImageCats = styled.Image`
+flex:1;
+width: null;
+height: 300;
+left: 0; 
+right: 0; 
+marginLeft:0; 
+marginRight:0; 
+marginTop: 10; 
+marginBottom: 10;
```

Comment:
indentation 😉

also, remember when you use styled components you write the css as "usual" so not `marginLeft` but `margin-left`